dndtroops-persistence/src/main/java/cz/muni/fi/pa165/dndtroops/entities/Troop.java

```diff
@@ -1,7 +1,128 @@
 package cz.muni.fi.pa165.dndtroops.entities;
 
+import javax.persistence.*;
+import java.util.Objects;
+
+/*
+
+Entity class for entity Troop.
+Creted by: Vojtech Duchon (UCO:410007)
+
+ */
+
+
+@Entity
+@Table (name = "troop")
 public class Troop {
 
-    // TODO
+
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    @Column (name= "id")
```

Comment:
Column (name =...) is unnecessary too. Remeber the JPA monster is reflecting your code into the database.